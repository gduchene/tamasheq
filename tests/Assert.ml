let x =
  assert (1 > 0);
  0
