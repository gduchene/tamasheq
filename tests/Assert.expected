Assert_0	=	[1] {
    (no name)	=	0
}
